### PR TITLE
Generate MySQL docs based on v0.18.6 of the pulumi-mysql provider

### PR DESCRIPTION
**Pull request by stack72**

### CODE REVIEW

File: content/docs/reference/pkg/python/pulumi/_index.md [hunk 1]
```diff
@@ -226,24 +220,11 @@ provider is pulled from the parent’s provider bag.</li>
 <li><strong>providers</strong> (<em>Optional</em><em>[</em><em>Mapping</em><em>[</em><em>str</em><em>,</em><a class="reference internal" href="#pulumi.ProviderResource" title="pulumi.ProviderResource"><em>ProviderResource</em></a><em>]</em><em>]</em>) – An optional set of providers to use for child resources. Keyed
 by package name (e.g. “aws”)</li>
 <li><strong>delete_before_replace</strong> (<em>Optional</em><em>[</em><em>bool</em><em>]</em>) – If provided and True, this resource must be deleted before it is replaced.</li>
-<li><strong>ignore_changes</strong> (<em>Optional</em><em>[</em><em>List</em><em>[</em><em>string</em><em>]</em><em>]</em>) – If provided, a list of property names to ignore for purposes of updates
```
Comment (justinvp) [inline content/docs/reference/pkg/python/pulumi/_index.md]: Do you know why these were removed?

Comment (stack72) [reply content/docs/reference/pkg/python/pulumi/_index.md]: I do not! Nothing has changed

Comment (stack72) [reply content/docs/reference/pkg/python/pulumi/_index.md]: on the package side of things I mean

**Comment by stack72** [comment]
Going to close this out and regenerate

**Pull request closed without merging**